src/main/scala/io/iohk/ethereum/jsonrpc/Web3Service.scala

```diff
@@ -25,4 +35,24 @@ class Web3Service {
     Future.successful(ClientVersionResponse(Config.clientVersion))
   }
 
+  def submitHashRate(req: SubmitHashRateRequest): Future[SubmitHashRateResponse] = {
```

Comment:
@AlanVerbner ok, so should I move this to EthService.scala now or after your PR merge?